### PR TITLE
Creat remove books

### PR DESCRIPTION
**Pull request by den-cloud**
> ## Here is what has been done to configure your Redux Store and write actions and reducer for adding and removing books.
> * Configuring the Redux Store (/src/redux/configureStore.js) and importing the necessary methods from Redux.
> * Write your book's actions and reducer (in /src/redux/books/books.js) using the ducks pattern.
>   
>   * Define action types
>   * Set initial state to be an empty array of books.
>   * Export your Action Creators (functions, which return actions)
>   * Write your reducer and export it as default.
> * In your React component responsible for adding new books set the data inputs in the local React state.
> * Implement that even by dispatching a corresponding action.
> * Use the redux-logger to check if your application is working correctly.
> * Update README.md file.

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+import booksReducer from './books';
+import filterReducer from './filter';
```
Comment (mikethreels) [inline src/reducers/index.js]: - [ ] according to the requirements in this component you should Import `createStore`, `combineReducers`  and `applyMiddleware`  from `redux`

File: src/reducers/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+const rootReducer = combineReducers({ theBooks: booksReducer, filter: filterReducer });
```
Comment (mikethreels) [inline src/reducers/index.js]: - [ ] according to the requirements you should use `createStore` to create a store through the reducer created in the above step.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Denzel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @den-cloud, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**